examples/CustomDropdown/CustomDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, createContext, use, useCallback } from "react";
+
+import { DayPicker, DropdownProps } from "react-day-picker";
+
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+  SelectStyles
+} from "./Select";
+
+const ContainerContext = createContext<HTMLDivElement | null>(null);
+
+export function CustomSelectDropdown(props: DropdownProps) {
+  const { options, value, onChange, "aria-label": ariaLabel } = props;
+  const container = use(ContainerContext);
+
+  const handleValueChange = (newValue: string) => {
```

Comment:
The idea is to add a `onValueChange` to the props:

```suggestion
  const { options, value, onValueChange, "aria-label": ariaLabel } = props;
  const container = use(ContainerContext);

  const handleValueChange = onValueChange
```